app/code/Magento/Cms/Test/Mftf/ActionGroup/AdminCreateNewCMSBlockActionGroup.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminCreateNewCMSBlockActionGroup">
```

Comment:
Can we create the block using an entity instead of introducing the new action group? Please, check the following example 

https://github.com/magento/magento2/blob/bba6a438c71045eef0083bfa3374065b26d9485d/app/code/Magento/Cms/Test/Mftf/Test/AdminCheckCreateFolderEscapeAndEnterHandlesForWYSIWYGBlockTest.xml#L23

https://github.com/magento/magento2/blob/bba6a438c71045eef0083bfa3374065b26d9485d/app/code/Magento/Cms/Test/Mftf/Test/AdminCheckCreateFolderEscapeAndEnterHandlesForWYSIWYGBlockTest.xml#L35

Reply:
**Done**